core/prelude-core.el

```diff
@@ -285,6 +285,24 @@ there's a region, all lines that region covers will be duplicated."
   (interactive)
   (byte-recompile-directory prelude-dir 0))
 
+(defun prelude-file-uid (filename)
+  "Return the UID of the FILENAME, normally as a string.
+
+See `file-attributes for more info.'"
+  (nth 2 (file-attributes filename)))
+
+(defun prelude-file-owned-by-user-p (filename)
+  "Return t if file FILENAME is owned by the currently logged in user."
+  (let ((uid (prelude-file-uid filename)))
+    (or (eq uid
```

Comment:
From reading the documentation I got the impression that the file attribute uid may sometimes come back as a string representing the username.  If that's not the case then we don't need two checks.